File: LeetCode/Missing_Number.cpp
/*
Problem Link:
https://leetcode.com/problems/missing-number/description/
*/
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int len =  nums.size();
        int sum = 0;
        for(int iter =0;iter<len;iter++){
            sum += nums[iter];
        }
        return ((len*(len+1))/2)-sum;
    }
};

File: LeetCode/missing_number.java
class Solution {
    public int missingNumber(int[] nums) {
        int missingNumber = 0;
        for(int iter = 0;iter< nums.length; iter++){
            missingNumber = missingNumber ^ nums[iter];
            missingNumber = missingNumber ^ iter;
        }
        return missingNumber^nums.length;
    }
}